multimodal/pyrightconfig.json

```diff
@@ -1,8 +1,5 @@
 {
```

Comment:
I think we can remove this entire file, right? Since we chose not to use pyright for the multimodal package.

Reply:
Yes, but let's please leave it for a future PR.